backend/uasmartsignage/sonar-project.properties

```diff
@@ -0,0 +1,4 @@
+sonar.projectKey=ua-smart-signage-platform_backend
+sonar.organization=ua-smart-signage-platform
+sonar.sources=.
+sonar.java.binaries=target/classes
```

Comment:
Is this needed when it is mentioned in the POM?

Reply:
The code that is in pom.xml file is enough and this is not needed.

Reply:
I erased this file as it was not necessary.